packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -88,6 +89,13 @@ export const sliderStyles: FoundationElementTemplate<ElementStyles, SliderOption
     .thumb-cursor:active {
         background: ${neutralForegroundRest};
     }
+    .track-start {
+        background: ${accentForegroundRest};
+        position: absolute;
+        height: 100%;
+        left: 0;
```

Comment:
Does the usage of the `left` property here need to change in RTL documents?

Reply:
Yep, good catch. Resolved.